xoa_driver/functions/anlt_ll_debug.py

```diff
@@ -117,7 +118,7 @@ async def __set(self, reg, value) -> None:
 
     lt_rx_analyzer_rd_data_get = partialmethod(__get, reg=LT_RX_ANALYZER_RD_DATA)
 
-    async def lt_prbs(self) -> Dict[str, int]:
+    async def lt_prbs(self) -> Dict[str, Union[int, float]]:
```

Comment:
if to return a dict is not a must then better to chose, Dataclass or named tuple, if you prefer a dict then use TypedDict for type notation, coz here we having a fixed fields